### ISSUE TITLE
R shows Error in loadSQM(): An error occurred while running sqm2tables.py

### ISSUE THREAD

**Issue by marius44** [jtamames/SqueezeMeta#344]
Hi meta-community!

I ran SqueezeMeta without problems, and I got the results. I did it using a server that runs under Ubuntu 20.04. On the same server I ran sqm2tables.py, everything ok. I downloaded the whole results folder into a personal Windows 10 machine for further SQMTools analysis. As I saw on other issues, I didn't change any name.

After many attempts (in many computers) I realized that SQMTools (Version 0.4.5) seems to work well using R 3.6.2 (It was kind of impossible to install it or to load it in R 4.1.0). When I try to make my SQM object, I get this error.

```
 # Making SQM object
> MB <- loadSQM("C:/Users/mario/Desktop/MB/")
Generating tabular outputs for project in C:/Users/mario/Desktop/MB/
Error in loadSQM("C:/Users/mario/Desktop/MB/") : 
  An error occurred while running sqm2tables.py
```

 Find attached the pl archives I saw you need for more information.

Best,

Mario.

[methods.zip](https://github.com/jtamames/SqueezeMeta/files/6948184/methods.zip)
[parameters.zip](https://github.com/jtamames/SqueezeMeta/files/6948185/parameters.zip)
[progress.zip](https://github.com/jtamames/SqueezeMeta/files/6948186/progress.zip)
[SqueezeMeta_conf.pl.zip](https://github.com/jtamames/SqueezeMeta/files/6948187/SqueezeMeta_conf.pl.zip)
![Error SQMTools](https://user-images.githubusercontent.com/47364310/128578285-f9e68057-84ee-407e-9db0-ecdc54b10b80.png)

**Comment by fpusan** [comment]
Hi!
If you ran `sqm2tables.py` back in your server it shouldn't be trying to run it in your personal machine.
What is the content of the `results` directory? Is the `results/tables` dir in there?

**Comment by marius44** [comment]
I don't get why does it try to run `sqm2tables.py` again. 

Inside `results` I got the results of the SqueezeMeta analysis:

01.MB.fasta
02.MB.16S.txt
02.MB.rnas
02.MB.trnas
02.MB.trnas.fasta
03.MB.faa
03.MB.fna                       
03.MB.gff
06.MB.fun3.tax.noidfilter.wranks
06.MB.fun3.tax.wranks
07.MB.fun3.cog
07.MB.fun3.kegg                  
07.MB.fun3.pfam                  
11.MB.mcount
12.MB.cog.funcover
13.MB.orftable
19.MB.bintable
20.MB.contigtable
21.MB.kegg.pathways
21.MB.metacyc.pathways
22.MB.stats
DAS/
maxbin/
metabat2/

I renamed `tables` generated by `sqm2tables.py` as `MB_sqm_tables`. If you're asking for that directory, I assume it should be inside `results`, Is there any problem with the name I gave it? Should I rename the directory to `tables`?

Best,

Mario.
